taskmgr/storage/task_store_db.go

```diff
@@ -55,8 +42,9 @@ func (db *PostgresDatabase) GetTasks() []*task.Task {
 
 	for rows.Next() {
 		r := &task.Task{}
+		var author string
 
-		err := rows.Scan(&r.ID, &r.Title, &r.Desc)
+		err := rows.Scan(&r.ID, &r.Title, &r.Desc, &author)
```

Comment:
since we are not returning the author later on, the AuthorID will always be 00000000-0000-0000-0000-000000000000. Is there a specific reason why you are not using the AuthorID in the task struct?
```suggestion
		err := rows.Scan(&r.ID, &r.Title, &r.Desc, &r.AuthorID)
```

Reply:
Yep this is also intentional, the author id should be taken from the author that created the task, but so far there is no logic to bind the two things togehter, but we have all the pieces to do it
we just need to do it.